### ISSUE TITLE
Notify in server-tab on disconnect

### ISSUE THREAD

**Issue by sebsel** [thelounge/thelounge#876]
I like to hide all the joins and quits, keeps the chat clean. But when I get disconnected, I would like to see my own quit-message, so that I know I'm out.

Right now the only way to see that I quit is in the server-tab, but that doesn't message me. So I've been disconnected for a few hours without noticing. If the server-tab would notify me on disconnect that would also solve my problem :)

**Comment by YaManicKill** [comment]
Now, this is an issue, but I don't think that's a good way to deal with it. Showing something that the user has explicitly requested not to be shown is a bad idea.

But we can show something to show that you are disconnected from the server.

However, what version of the lounge are you on just now? Is it not auto-reconnecting?

**Comment by sebsel** [comment]
I do agree on that! A message that you are disconnected is maybe the best way, because in order to reconnect, you have to be on that same tab anyway.

I had this, I don't know if that's auto-reconnecting:
```
04:02		Ping timeout, disconnecting…
04:03		Closing Link: [...host...] (Ping timeout: 245 seconds) (irc)
04:03		Disconnected from the network, and will not reconnect. Use /connect to reconnect again.
```

I'm on 2.1.0, that's the latest right?

**Comment by xPaw** [comment]
Seems as easy as adding :not(.self) to selectors.

**Comment by sebsel** [comment]
Yeah, there is also this: https://github.com/thelounge/lounge/blob/master/client/js/lounge.js#L109

Apparently the send-button hides there. One could also add a class to `#input` and make it red or grey, that would also signify that you're not connected. (I didn't spot the hidden send-button, it's too minor)

**Comment by xPaw** [comment]
This was fixed in Vue because it never condenses or hides messages that affect you (joins, kicks, quits, etc).

**Comment by xPaw** [comment]
Oops, I actually just looked at the code, that only applies to condensed, it still hides all status messages.
